### PR TITLE
[update] HowTo quick start AXC F 2152 and PLCnext Engineer

### PR DESCRIPTION
**Pull request by MrSoulfinder**

### CODE REVIEW

**Comment by idzm** [comment]
The latest LTS version is `2021.0.5_LTS`, not `2021.0.9_LTS`.

**Review by MrSoulfinder — COMMENTED** [review]
Change the last version

File: HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The firmware latest release is located [here](https://www.phoenixcontact.com/onl
 
 To update the controller firmware, proceed as follows:
 
-- Download the *.zip firmware file with version **2020.0.1 LTS** at Phoenix Contact [site](https://select.phoenixcontact.com/phoenix/dwl/dwl13a.jsp?fct=dwl&asid=7449344&name=AXC_F_2152_FW2020_0_1_LTS.zip&prodid=2404267&lang=en&hwv=&debug=0).
+- Download the *.zip firmware file with version **2021.0.5 LTS** at Phoenix Contact [site](https://select.phoenixcontact.com/phoenix/dwl/dwl13a.jsp?fct=dwl&asid=7449344&name=AXC_F_2152_FW2020_0_1_LTS.zip&prodid=2404267&lang=en&hwv=&debug=0).
```
Comment (idzm) [inline HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md:31]: You also should change link to version `2021.0.5`.

File: HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Following this, the controller is restarted. Once the controller has been fully
 
 ## PLCnext Engineer project creation ##
 
-It's important to use the appropriate version of **`"PLCnext Engineer"`** - **`"2020.0.1"`**.  Start **`"PLCnext Engineer"`** and select **`"Empty AXC F 2152 v00 / 2020.0.0 project"`**:
+It's important to use the appropriate version of **`"PLCnext Engineer"`** - **`"2021.0.5"`**.  Start **`"PLCnext Engineer"`** and select **`"Empty AXC F 2152 v00 / 2021.0.5 project"`**:
```
Comment (idzm) [inline HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md:77]:
```suggestion
It's important to use the appropriate version of **`"PLCnext Engineer"`** - **`"2021.0.5"`** (or newer).  Start **`"PLCnext Engineer"`** and select **`"Empty AXC F 2152 v00 / 2021.0.5 project"`**:
```

**Review by idzm — CHANGES_REQUESTED** [review]

**Review by MrSoulfinder — COMMENTED** [review]
update link for version 2021.0.5

File: HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Following this, the controller is restarted. Once the controller has been fully
 
 ## PLCnext Engineer project creation ##
 
-It's important to use the appropriate version of **`"PLCnext Engineer"`** - **`"2020.0.1"`**.  Start **`"PLCnext Engineer"`** and select **`"Empty AXC F 2152 v00 / 2020.0.0 project"`**:
+It's important to use the appropriate version of **`"PLCnext Engineer"`** - **`"2021.0.5"`** (or newer).  Start **`"PLCnext Engineer"`** and select **`"Empty AXC F 2152 v00 / 2021.0.5 project"`**:
```
Comment (idzm) [inline HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md:77]: We only have this option: 
```suggestion
It's important to use the appropriate version of **`"PLCnext Engineer"`** - **`"2021.0.5"`** (or newer).  Start **`"PLCnext Engineer"`** and select **`"Empty AXC F 2152 v00 / 2021.0.0 project"`**:
```

**Review by idzm — CHANGES_REQUESTED** [review]

File: HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md (hunk 3 above)
Comment (MrSoulfinder) [reply HowTo quick start AXC F 2152 and PLCnext Engineer/readme.md]: understand, corrected

**Review by idzm — CHANGES_REQUESTED** [review]
Archive `AXC_F_2152_FW2021_0_5_Bundle.zip` already contains `axcf2152-2021.0.5.35585-LTS.raucb` - so we do not need to run an installer.

**Review by MrSoulfinder — COMMENTED** [review]
Change the installation steps

**Review by idzm — APPROVED** [review]

**Pull request merged**